src/StudentPackage/TakingInputFromUser.java

```diff
@@ -0,0 +1,111 @@
+package StudentPackage;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.Comparator;
+public class TakingInputFromUser {
```

Comment:
Rename class 
Don't add verbs like Taking
Call it something like UserIO or InputOutput

Reply:
Done